conf.yaml

```
@@ -734,7 +734,8 @@ contents:
                 repo:   cloud-assets
                 path:   docs
                 map_branches: &mapCloudEceToCloudAssets
-                  release-ms-41: master
+                  ms-47: master
+                  2.6: master
```

```suggestion
                  release-ms-41: master
```
Same as above, we need to use the branch name here.